planet.js

```diff
@@ -739,35 +739,36 @@ const _connectRoom = async roomName => {
         const {method} = j;
         if (method === 'pose') {
           const {pose} = j;
-          const [head, leftGamepad, rightGamepad] = pose;
+          const [head, leftGamepad, rightGamepad, floorHeight] = pose;
 
-          console.log('got pose', head[0].join(','));
+          // console.log('got pose', head[0].join(','));
           window.remotePose = pose;
-          window.remoteRig = playerRig;
+          window.remoteRig = peerRig;
```

Comment:
Probably shouldn't latch to window anymore; this was for debugging.

Reply:
fixed